addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15904,7 +15914,7 @@ msgstr ""
 #. Description of setting "Videos -> Playback -> Enable MMAL hardware decoding of video files"
 #: system/settings/settings.xml
 msgctxt "#36434"
-msgid "Allow hardware acceleration - MMAL"
+msgid "Allow hardware acceleration · MMAL"
```

Comment:
The skin language uses these, I thought they look ok here also (they sort of do), but if you really want them back Ill put them back, no biggie